.changeset/small-cups-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': minor
+---
+
+Add 1 new icon: `homebrew`.
```

Comment:
Small tweak here — we landed on using third-person conjugation for our changesets, so “adds” rather than “add”:

```suggestion
Adds 1 new icon: `homebrew`.
```